nextcloud_install_production.sh

```diff
@@ -389,6 +389,8 @@ crontab -u www-data -l | { cat; echo "*/5  *  *  *  * php -f $NCPATH/cron.php >
 
 # Run the updatenotification on a schelude
 occ_command config:system:set upgrade.disable-web --value="true"
+occ_command app:disable updatenotification
+rm -r $NCPATH/apps/updatenotification
```

Comment:
Are you 100% this doesn't disable email notifications when there's a new notification in the GUI?

Reply:
As I stated here: https://github.com/nextcloud/vm/pull/1035#issuecomment-575159638

> > > occ app:disable updatenotification
> > 
> > 
> > Would that disable emails as well?
> 
> no, it shouldn't.
> I have run the snap for years with disabled update notifications and email notifications always worked
> 
> to be more specific:
> activity notifications with email always worked with the snap
> but if there are email notifications sent for nextcloud or app updates, then these would probably don't work.